knox/views.py

```diff
@@ -15,6 +18,12 @@ class LoginView(APIView):
     permission_classes = (IsAuthenticated,)
 
     def post(self, request, format=None):
+        if settings.REST_KNOX['TOKEN_LIMIT'] is not None:
```

Comment:
please have a look at my changes here https://github.com/James1345/django-rest-knox/commit/c747f40a43fb1bd2c713183b1a49afbbcfdf60d0
I think it is the cleaner way of accessing settings